README.md

```diff
@@ -121,6 +121,9 @@ Optional arguments:
     It is enforced on per-second basis, to avoid bursts.
 - `--traveltime-max-rpm [int]`: Set max number of parallel requests sent to TravelTime API per minute. Default is 60.
   It is enforced on per-second basis, to avoid bursts.
+- `--providers [providers]`: The providers you want to compare to TravelTime (e.g., --providers google mapbox).
```

Comment:
Should they be space separated like in the example? Would comma also work? I would clarify it

Reply:
I'll rework it with a config file anyways